mkosi/resources/mkosi.md

```diff
@@ -556,6 +556,18 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
   subdirectory of the workspace directory instead of the OS tree. This
   subdirectory of the workspace is used to configure the package manager.
 
+: mkosi will look for various package manager and related files in the
+  configured package manager trees. Unless specified otherwise, it will
+  look up a configuration file at its canonical location in the package
+  manager trees. (e.g. it will look for `dnf.conf` at
+  `etc/dnf/dnf.conf`) in the package manager trees.
```

Comment:
Backticks around `mkosi`.
Also "look up" is jargony. Just "read … from" or "load … from" or "use" would be better.

Maybe:
> `mkosi` will look for the package manager configuration and related files in the configured package manager trees. Unless specified otherwise, it will use the configuration file from its canonical location in the package manager trees. For example, it would use `etc/dnf/dnf.conf` if the `rpm` package manager is used.